chart/kubeapps/templates/kubeappsapis/rbac_fluxv2.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.featureFlags.kubeappsAPIsServer }}
+{{- if has "fluxv2" .Values.kubeappsapis.enabledPlugins }}
+{{- if .Values.rbac.create -}}
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+kind: ClusterRole
+metadata:
+  name: "kubeapps:controller:kubeapps-apis-fluxv2-plugin"
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: kubeappsapis
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" . ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+rules:
+  - apiGroups: ["source.toolkit.fluxcd.io", "helm.toolkit.fluxcd.io"]
+    resources: ["helmrepositories", "helmcharts", "helmreleases"]
+    verbs: ["get", "list", "watch", "create", "update", "delete"]
```

Comment:
Just want to verify that the plugin really does need all 6 verbs? I'm surprised only because I thought the user is creating the HelmRelease CR.

Reply:
Nope, the plug-in creates HelmRelease CRD when the user calls CreateInstalledPackage(...)